chia/full_node/block_store.py

```diff
@@ -58,7 +58,7 @@ async def create(cls, db_wrapper: DBWrapper):
         await self.db.execute("CREATE INDEX IF NOT EXISTS height on block_records(height)")
 
         await self.db.execute("CREATE INDEX IF NOT EXISTS hh on block_records(header_hash)")
-        await self.db.execute("CREATE INDEX IF NOT EXISTS peak on block_records(is_peak)")
+        await self.db.execute("CREATE INDEX IF NOT EXISTS peak on block_records(is_peak) where is_peak = 1")
```

Comment:
this change is covered by another PR. In order to allow transitioning existing users/databases, I think the new index needs a new name. Otherwise it won't be created for existing users.

Reply:
What's the other PR? If it's covered already, we can skip this one. I was using the same name, as I didn't think it made sense to have both indices created, so only new installation would benefit.  Existing users are really only benefitting, if the old index didn't exists e.g. was dropped.

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/9349